tailwind.config.js

```diff
@@ -1,5 +1,11 @@
-const variableTokens = require('./src/tokens-dist/json/css-variables-nested.json');
-const staticTokens = require('./src/tokens-dist/json/variables-nested.json');
+const edsTokens = require('./lib/tokens/json/css-variables-nested.json').eds;
```

Comment:
This is needed b/c `lib` is available in module but not `src`, but this causes CI tests to fail since `lib` is not being created in CI. How can I fix the CI tests? Add `src/tokens-dist/json/css-variables-nested.json` to files and require that? Build `lib` in CI? Alternatives?

Reply:
Mostly already built in the ci runs, tokens just now has to be built before lint job since tw config references it and its absence triggers lint error